### DISCUSSION TITLE
Discord & Game Account Banned w/o a word

### DISCUSSION THREAD

**Discussion by Daedalus007** [chromiecraft/chromiecraft#6811] [General]
Seems my Discord and Chromiecraft game accounts were both banned without a word/reason.
I'd like to know the reason behind the bans at the least, as I haven't played the game for 3+ years now after going on an extended haitus from WoW in general.  Been enjoying some other games and wanted to hop back to see how things were going.  Surprise surprise.

So trying to login to the game gives me the 'game account banned' error message.  No details on why or how.  I didn't play the account for a while and I can still login via the website/wordpress login.

I tried to access the Discord server and it gives me the 'invite invalid' link aka 'you've been banned but we won't even tell you' as Discord tends to do.  Since there is no ban ticket/appeal/etc system available via the website, people like myself and many others have to randomly stumble here and put our info out in public just to get a response.

Fortunately I'm not caring all that much about exact details.  My info for game account and Discord should be easy enough to find as I tend to use this same github username as my identity online across various services like Steam.

**Comment by Daedalus007** [comment]
So I suppose TLDR:  tell me why my game account was banned despite not violating any rules and why my Discord account was banned w/o a word; also unban both plz as it has been 3+ years and whatever salt happened needs to be dried up by now.  Kthx.

**Comment by 55Honey** [comment]
Hello @Daedalus007 
You have been banned permanently from our game server in October 2021. You're not welcome to return. Permanent means infinite ban duration.

Good luck someplace else. Regards

**Reply by Daedalus007** [reply]
Kinda amusing that you ban people without telling them why.   I suppose if you want to speedrun the server into the dirt you're doing a fantastic job!
'regards'

**Reply by 55Honey** [reply]
Your ban reads "insulting GMs and players"

Thank you for the feedback.

**Reply by Daedalus007** [reply]
Your GM at the time (3 years ago, mind you) was abusing GM powers by randomly teleporting newbies around the starting areas (I was in Ghostlands at the time and saw this firsthand) as well as spewing sexual insults against female characters (particularly female blood elves).  When I called them out on this and asked them to stop, they banned me.

Of course knowing your infamous reputation 'Honey', I'm not surprised.  Hope that $50 profit a month is making it worth being the ruler of an empty WoW server.

**Reply by 55Honey** [reply]
I am not surprised that I have a reputation among those who break the rules. That indicates I fulfilled my role just fine.
Anyway, blaming anyone for anything won't bring your account back.
